FlowCrypt/src/main/java/com/flowcrypt/email/service/attachment/AttachmentDownloadManagerService.kt

```diff
@@ -652,6 +635,27 @@ class AttachmentDownloadManagerService : LifecycleService() {
         put(MediaStore.Downloads.IS_PENDING, 0)
       }, null, null)
 
+      resolver.query(
+        fileUri,
+        arrayOf(MediaStore.DownloadColumns.DISPLAY_NAME),
+        null,
+        null,
+        null
+      )
+        ?.use { cursor ->
+          //we should check maybe a file is already exist.
+          //Then we will use the file name from the system.
+          if (cursor.moveToFirst()) {
+            val nameIndex = cursor.getColumnIndex(MediaStore.DownloadColumns.DISPLAY_NAME)
```

Comment:
I think better to use `getColumnIndexOrThrow` to avoid checking `-1` and fails fast if the column doesn't exist?